pkg/os/os-mapper-provider.go

```diff
@@ -53,22 +54,20 @@ func (o *OSMaps) GetOSMaps() (map[string]string, map[string]string, error) {
 
 func (o *OSMaps) updateOsMapsByUserMaps(guestOsToCommon map[string]string, osInfoToCommon map[string]string) error {
 	osConfigMap := &corev1.ConfigMap{}
-	osConfigMapName := os.Getenv(OsConfigMapName)
-	osConfigMapNamespace := os.Getenv(OsConfigMapNamespace)
-	if osConfigMapName == "" && osConfigMapNamespace == "" {
+	if o.osConfigMapName == "" && o.osConfigMapNamespace == "" {
 		return nil
 	}
-
-	err := o.Client.Get(
+	log.Info("Using user-provided OS maps", "name", o.osConfigMapName, "namespace", o.osConfigMapNamespace)
```

Comment:
just a note - this is a generic message not tied to any particular vm-import request.
but it seems enough to me, since we don't expect too frequent changes to it.